t/plugin/limit-count-redis.t

```diff
@@ -82,6 +82,10 @@ GET /t
     location /t {
         content_by_lua_block {
             local t = require("lib.test_admin").test
+            -- SET redis instance password
+            local tmp_auth = "foobared"
+            local val = redis.pcall('CONFIG', 'SET', 'requirepass', tmp_auth)
```

Comment:
After running the test case, we need to restore the Redis state. Other plugins maybe need to use the Redis too.